### ISSUE TITLE
Migrate to http://rollup.js.org/

### ISSUE THREAD

**Issue by Victorystick** [rollup/rollupjs.org#22]
I've just read about https://js.org/. They offer free subdomains for JS projects, so we could ask for `rollup.js.org`. It's just a PR away. I thought it might be nice not having to pay for/manage it yourself. :smile:

**Comment by Rich-Harris** [comment]
Yeah, I wondered about js.org before I bought the domain (which was cheap! only $12 for the year) but I decided against it at the time for a couple of reasons:

* they don't have a paid version of the service, so I don't trust it to still exist in years to come! I can't believe *that* many people are buying JavaScript mugs
* I've had problems with GitHub pages in the past, and js.org locks you into them – surge.sh is faster on deploy, more flexible, and more fully featured

That said it might make sense to claim the subdomain and set it up as a redirect...
